applications/vanilla/modules/class.tagmodule.php

```diff
@@ -103,6 +103,15 @@ public function getData() {
 
         $this->autoContext();
 
```

Comment:
Somewhere in here I feel like an inline comment would be appropriate about why this change is being made. "// Allow an addon to hijack the setting of our data, which aborts the normal database queries." or something to that effect.